### PR TITLE
Clarify difference between only get accessor and get with private set

### LINKED ISSUE #16807: The example does not reveal the essence of "private set;" # 248
@vctrro commented on [Wed Jan 22 2020](https://github.com/dotnet/docs.ru-ru/issues/248)

In this example, there is no difference between {get; } and {get; private set; }

---
#### Сведения о документе

⚠ *Не вносите правки в этот раздел. Это необходимо для связывания страницы сайта docs.microsoft.com с вопросом на GitHub.*

* ID: d549e670-9c25-f742-f146-699ba15e39e5
* Version Independent ID: 88fce03f-2ce8-0dd6-ae04-4676fcbb864d
* Content: [Практическое руководство. Реализация облегченного класса с автоматически реализуемыми свойствами (руководство по программированию на C#)](https://docs.microsoft.com/ru-ru/dotnet/csharp/programming-guide/classes-and-structs/how-to-implement-a-lightweight-class-with-auto-implemented-properties#feedback)
* Content Source: [docs/csharp/programming-guide/classes-and-structs/how-to-implement-a-lightweight-class-with-auto-implemented-properties.md](https://github.com/dotnet/docs.ru-ru/blob/live/docs/csharp/programming-guide/classes-and-structs/how-to-implement-a-lightweight-class-with-auto-implemented-properties.md)
* Product: **dotnet-csharp**
* Technology: **csharp-fundamentals**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

---

@srvbpigh commented on [Wed Jan 22 2020](https://github.com/dotnet/docs.ru-ru/issues/248#issuecomment-577068547)

Hello, @vctrro
 
Thank you for your feedback.
 
We are actively reviewing your comments and will get back to you soon.

Kind regards,
Microsoft DOCS International Team

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes #16807

### CODE REVIEW

File: docs/csharp/programming-guide/classes-and-structs/how-to-implement-a-lightweight-class-with-auto-implemented-properties.md [hunk 1]
```diff
@@ -27,7 +50,7 @@ The following example shows two ways to implement an immutable class that has au
 // constructor to initialize its properties.
 class Contact
 {
-    // Read-only properties.
+    // Read-only property.
     public string Name { get; }
     public string Address { get; private set; }
```
Comment (Youssef1313) [inline docs/csharp/programming-guide/classes-and-structs/how-to-implement-a-lightweight-class-with-auto-implemented-properties.md]: I don't think the "Address" property is considered as read-only. Only "Name" is read-only. So, I changed "Read-only properties" to "Read-only property".

Comment (BillWagner) [reply docs/csharp/programming-guide/classes-and-structs/how-to-implement-a-lightweight-class-with-auto-implemented-properties.md]: In this instance, we need to clarify the subtlety:

From the perspective of the public interface, `Address` is a read-only property. In contrast, from the perspective of immutability and members of the type, `Address` is not read-only.

Comment (BillWagner) [reply docs/csharp/programming-guide/classes-and-structs/how-to-implement-a-lightweight-class-with-auto-implemented-properties.md]: I think the best way to highlight this distinction is to refer to one as "read-only" property, and the other as a "read-write property with a private set accessor"

It is more wordy, but it is consistent with the spec language.

**Review by BillWagner — CHANGES_REQUESTED** [review]
Overall, like the changes you've made @Youssef1313 

I have a few small suggestions. After those updates, it's ready to :shipit:

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
thanks for making these updates @Youssef1313 

I'll :shipit: now.

**Pull request merged**